app/api/helpers/storage.py

```diff
@@ -180,10 +182,16 @@ def upload_local(uploaded_file, key, **kwargs):
     return create_url(request.url, file_relative_path)
 
 
-def create_url(request_url, file_relative_path):
+def create_url(request_url, file_relative_path, is_test=False):
     """Generates the URL of an uploaded file."""
     url = urlparse(request_url)
-
+    if not is_test:
+        if app.config['SERVER_URL']:
+            url = urlparse(app.config['SERVER_URL'])
+        else:
+            pass
+    else:
+        pass
```

Comment:
Seems good, just one thing, is there no need to load test_Server_url  in this case?

Reply:
I hasn't able to find any reason to use that, for now.
Can you clear it if its really of use.

Reply:
There's no need to write these `pass` statements here.